src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,28 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div
+        className="card"
+        data-cy="Movie"
+        key={movie.imdbId}
+      >
+        <MovieCard
+          title={movie.title}
+          description={movie.description}
+          imgUrl={movie.imgUrl}
+          imdbUrl={movie.imdbUrl}
+        />
```

Comment:
You can pass the whole movie and make destructuring inside MovieCard component to get all the needed props of movie